cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentLibBuilder.java

```diff
@@ -9,6 +9,8 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
+import java.util.concurrent.RunnableFuture;
```

Comment:
Was this added by mistake?

Reply:
yep, rookie mistake. got rid of it before merging though 👍